webapp/sass/layout/_post.scss

```diff
@@ -377,6 +377,7 @@ body.ios {
         }
 
         .custom-textarea {
+            bottom: 0;
```

Comment:
Did you check that the @mention autocomplete still appears in the right place with this? I think I removed it to fix that